packages/serverless-components/domain/utils.js

```diff
@@ -408,19 +408,24 @@ const addDomainToCloudfrontDistribution = async (
   params.Id = subdomain.distributionId;
 
   // 5. then make our changes
-  params.DistributionConfig.Aliases.Items.push(subdomain.domain);
+  params.DistributionConfig.Aliases = {
+    Quantity: 1,
```

Comment:
No real reason to set this to 1, it'll get set correctly on line 429.

Reply:
Sure, I can remove it, thanks.